snapshots.nix

```diff
@@ -6,15 +6,22 @@
 # A particular package in a snapshot would be accessed with:
 #   snapshots."lts-13.18".conduit
 
-{ lib, mkPkgSet, stackage }:
+{ lib, mkPkgSet, stackage, ghc-boot-packages }:
 
 with lib;
 
 let
   mkSnapshot = name: pkg-def: (let pkgSet = mkPkgSet {
-    inherit pkg-def;
-    pkg-def-extras = pkg-def-extras name;
+    # libiserv is not in hackage so we should not try to get it from there
+    pkg-def = hackage:
+      let original = pkg-def hackage;
+      in original // {
+        packages = lib.filterAttrs (n: _: n != "libiserv") original.packages;
```

Comment:
Is it only libiserv? It would we want to strip everything that’s in the ghc-boot-packages? If not , why? Should add a comment why/why not.